api/db-cdr/generate-cdr/make-bq-data.sh

```diff
@@ -177,9 +177,13 @@ echo "Inserting concept table data ... "
 bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
 "INSERT INTO \`$WORKBENCH_PROJECT.$WORKBENCH_DATASET.concept\`
 (concept_id, concept_name, domain_id, vocabulary_id, concept_class_id, standard_concept,
-concept_code, count_value, prevalence, source_count_value)
+concept_code, count_value, prevalence, source_count_value, synonyms)
+with concept_synonyms as
+(select c.concept_id as concept, concat(cast(c.concept_id as string),'|',c.concept_code,'|',c.concept_name,'|',string_agg(cs.concept_synonym_name,'|')) as synonyms
+from \`${BQ_PROJECT}.${BQ_DATASET}.concept\` c join \`${BQ_PROJECT}.${BQ_DATASET}.concept_synonym\` cs
+on c.concept_id=cs.concept_id group by c.concept_id,c.concept_code,c.concept_name)
 SELECT c.concept_id, c.concept_name, c.domain_id, c.vocabulary_id, c.concept_class_id, c.standard_concept, c.concept_code,
-0 as count_value , 0.0 as prevalence, 0 as source_count_value
+0 as count_value , 0.0 as prevalence, 0 as source_count_value,(select synonyms from concept_synonyms where concept=c.concept_id)
```

Comment:
Is it tricky to just do the synonym concat in this select rather than making the temp table ?  .. I also like joins over inner selects .. i think they perform better ... not sure though.  I would just move the join above and synonym generation here .

Reply:
If this works, not sure why we'd want a temp table? 

We do need to make sure this query returns just one row for each concept, so a join to concept synonym outside of a subselect probably doesn't work.

Reply:
Sorry. I call the " with concept_synonyms as..." thing above a temp table.  'm not sure why we just don't do it all in one query and  left join to concept_synonm table  and put hat concat statement where the select statement is.

Reply:
Added concat in the same query.

Reply:
So you'd do a left join and a group by?

That sounds fine; I'm not sure whether it makes much difference.